### DISCUSSION TITLE
Test exercises pygimli

### DISCUSSION THREAD

**Discussion by makeabhishek** [gimli-org/pyGIMLi#562] [General]
I saw the post #352, where it was mentioned that Florian is providing exercise next week. I'm wondering if there is a channel where we can learn more about #pygimli and if there are tutorials to get better understanding?

**Comment by carsten-forty2** [comment]
The post #352 was from Nov 30, 2021 so "next week" is kind of past.

You can take a look at this Tutorial: Geophysical modeling & inversion with pyGIMLi: https://www.youtube.com/watch?v=w3pu0H3dXe8
